NuKeeper/NuGet/Process/NuGetUpdater.cs

```diff
@@ -14,10 +15,10 @@ public NuGetUpdater(IExternalProcess externalProcess = null)
             _externalProcess = externalProcess ?? new ExternalProcess();
         }
 
-        public async Task UpdatePackage(PackageUpdate update)
+        public async Task UpdatePackage(PackageIdentity newPackage, PackageInProject currentPackage)
```

Comment:
I don't like that newPackage and currentPackage are different types - seems misleading.

Is there a reasonable renaming that can be done to alleviate that?

`newPackage` -> `newPackageIdentity`?